setup.py

```diff
@@ -21,7 +21,8 @@
     ''',
     data_files=[("share/colin/config/", ["config/default.json",
                                          "config/fedora.json",
-                                         "config/redhat.json"])],
+                                         "config/redhat.json"]),
+                ("share/bash-completion/completions/", ["colin-bash-complete"])],
```

Comment:
I liked just `colin` more. None of the files in `/usr/share/bash-completion/completions` have `bash` or `complete` in name.

Reply:
Yes, the previous version creates the `colin` directory in `..../completions` and place there the file.

Is there some way, how to rename the file in setup.py, or I have to name it colin in the repository?